# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../dietlibc/fenv.h.patch
# Copyright (C) 2011 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

Unfortunately latest libgcc started to utlize these.

  - Rene Rebe <rene@exactcode.de>

--- /dev/null	2010-11-17 09:26:17.638732721 +0000
+++ dietlibc-0.32/include/fenv.h	2011-01-06 14:53:10.774797870 +0000
@@ -0,0 +1,14 @@
+/* stubs, for gcc */
+#define FE_INEXACT	0x20
+#define FE_UNDERFLOW	0x10
+#define FE_OVERFLOW	0x08
+#define FE_DIVBYZERO	0x04
+#define FE_INVALID	0x01
+
+#define FE_ALL_EXCEPT \
+	(FE_INEXACT | FE_DIVBYZERO | FE_UNDERFLOW | FE_OVERFLOW | FE_INVALID)
+
+#define FE_TONEAREST	0
+#define FE_DOWNWARD	0x400
+#define FE_UPWARD	0x800
+#define FE_TOWARDZERO	0xc00
